### ISSUE TITLE
alpha-methylhistidine without stereo

### ISSUE THREAD

**Issue by smoretti** [ebi-chebi/ChEBI#4203]
Hi

We wonder why CHEBI:70960 (alpha-methylhistidine) has no stereo but its name is alpha...

**Comment by smoretti** [comment]
So CHEBI:70960 does not match with its Xref CPD-10906

**Comment by amalik01** [comment]
yes, you're right. The structure which MetaCyc are showing is different to the one present in ChEBI. Metacyc are showing the structure of alpha-methyl-L-histidine but have called it alpha-methyl histidine since they are assuming that the biologically active form will be the L-amino acid form. The major issue with alot of databases is that they are using ambiguous names for chemicals. Lets take histidine as an example, there are several histidine forms that can be synthetically synthesized (L-histidine, D-histidine, DL-histidine, and histidine where the stereochemistry is unspecified), and which form is one referring to. This is difficult to know unless you specifically specify the histidine form used.

I have created a separate ChEBI entry for the L-form (CHEBI:189415, will be searchable on the ChEBI website by tomorrow) and moved the MetaCyc Xref to this entry.
